lib/node_modules/@stdlib/math/base/special/riemann-zeta/package.json

```diff
@@ -67,5 +67,79 @@
     "special",
     "function",
     "complex analysis"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "zeta",
+      "alias": "zeta",
+      "pkg_desc": "evaluate the Riemann zeta function as a function of a real variable `s`",
+      "desc": "evaluates the Riemann zeta function as a function of a real variable `s`",
+      "short_desc": "Riemann zeta",
+      "parameters": [
+        {
+          "name": "x",
+          "desc": "input value",
+          "type": {
+            "javascript": "number",
+            "jsdoc": "number",
+            "c": "double",
+            "dtype": "float64"
+          },
+          "domain": [
+            {
+              "min": "-infinity",
+              "max": "infinity"
+            }
+          ],
+          "rand": {
+            "prng": "random/base/uniform",
+            "parameters": [
+              -100,
+              100
+            ]
+          },
+          "example_values": [
+            -10,
+            -5,
+            -4,
+            -3,
+            -2,
+            -1,
+            -0.5,
+            0,
+            0.5,
+            0.9,
+            1.1,
+            2,
+            3,
+            4,
+            5,
+            6,
+            10,
+            20,
+            50,
+            100
```

Comment:
@manvith2003 This sort of predictability is generally not desired. As functions often have differing behavior in different sections of their domain, such predictability can lead to "boring" examples, especially in those situations where we only use the first few example values for use in docs and examples.

Reply:
In short, I recognize the temptation to simply copy-paste from other packages. But I encourage you to take a minute or two longer to reflect on what makes sense on a per-package basis.